src/test/cypress/support/utils.ts

```diff
@@ -15,7 +15,7 @@ day.extend(utc);
  * */
 export function generateUUID() {
     const uuid = uuidv4().replace(/-/g, '');
-    return uuid.substr(0, 5);
+    return uuid.substr(0, 9);
```

Comment:
Are the UUIDs used anywhere else in some regexes etc? Just asking as this might be a problem then but guess not.

Reply:
No they are only used for unique entities within artemis